### ISSUE TITLE
Adding minimum and recommended disk size

### ISSUE THREAD

**Issue by mahnouman** [k8-proxy/go-k8s-infra#97]

**Comment by himanshu2464** [comment]
minimum & recommended disk size to be 40 GB.

**Comment by himanshu2464** [comment]
this is because when we hit the server for processing files with 50 threads the currently 20GB size is utilized like 90% so need to increase it to 35 GB.

**Comment by GINAGC** [comment]
@himanshu2464 is this already in readme file the information above?

**Comment by himanshu2464** [comment]
@GINAGC  yes it is already present in the readme file and I have added instructions to run a 10k file test with f2f.
https://github.com/k8-proxy/go-k8s-infra/pull/146

**Comment by GINAGC** [comment]
@Lestat06 please update the wiki with the recommended deployment for cloud sdk, since we are going to use always the minio version.

**Comment by Lestat06** [comment]
in the info below can we change the instance to t3.2xlarge? i know the cpu and ram is the same however after some readings it seems that T3 instances offer a balance of compute, memory, and network resources, and are designed to provide a baseline level of CPU performance with the ability to burst above that baseline when needed.

Instances functioning @ 100% for a 1-month period
T3.xlarge baseline performance is 1.6 vCPUs (40% * 4 vCPUs)
T2.xlarge  Baseline performance is 0.9 vCPUs (22.5% * 4 vCPUs)
Extra charges of $0.05 (for Linux) and $0.096 (for Windows) per vCPU-hour for T2/T3 bursting beyond credits

Quoting "T3 offers better value than T2, including extra CPU credits and better baseline performance, faster CPUs. The downside is the migration effort. The cost difference of the instance types between Windows and Linux is considerable and results in different optimal instance types: For Linux, T3 is priced lower than T2."

Moreover since the instructions on ck8s (which is the one that will be used) are with t3.2xlarge i think it will be confusing for the users to see different type of instance.
![image](https://user-images.githubusercontent.com/6232283/127409501-bf3945fd-919c-41cf-b373-8e546e5dc993.png)

**Comment by GINAGC** [comment]
@himanshu2464 please comment on Jurgen suggestion.

**Comment by dtollaku** [comment]
@GINAGC after Himanshu review, can we have this info on readme, so its not lost on the issue.

**Comment by GINAGC** [comment]
yes, we are also going to update on wiki @dtollaku

**Comment by Lestat06** [comment]
this is now added to wili

**Comment by himanshu2464** [comment]
@Lestat06 @dtollaku @GINAGC @mahnouman 
we can use this t3instance type family, so instead of t2.2xlarge, we can use t3.2xlarge.

**Comment by Lestat06** [comment]
thank you Himanshu. this is now added in wiki of the worker cluster integration

![image](https://user-images.githubusercontent.com/6232283/128082669-05ea41aa-9eb8-4f03-aa1e-0437d2d57702.png)
